pkg/util/containers/types.go

```diff
@@ -111,3 +111,15 @@ type ContainerImplementation interface {
 
 	metrics.ContainerMetricsProvider
 }
+
+// FilterType indicates the container filter type
+type FilterType string
+
+// Global is used to cover both Metrics and Logs filter types
+const Global FilterType = "Global"
```

Comment:
It is cosmetic but that looks weird to me to have global variables named `containers.Global`, `containers.Metrics` and `containers.Logs` actually being FilterType, don't you think? Not a big deal though.

I would have been with `containers.GlobalFilter`, `containers.MetricsFilter` and `containers.LogsFilter` I think and called `IsExcluded` -> `HasFilter` instead.

Reply:
good point, will do that thanks!